### ISSUE TITLE
Reformat readme structure

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-web-crawler#114]
@franciscawijaya , currently, the readme has six sections, but the content links at the top of the readme only have four, e.g., "Thank you" is now section 6.

Also, the "Components:" hangs a bit in the air on its own. This should probably be its own new section ("4. Components") or at least some subsection header.

Then, the whole part at the end starting with "Purpose of the well.known Python Script: analyze the full crawl set with the redo sites replaced" is still part of the second bullet point. If you can indent that to make it more clear. I was confused because I thought this is a new point. "Output:" etc should be formatted as headers etc.

Also, does the "This script is stored and executed locally. The crawler also keeps a log of sites that cause errors. It stores these logs in a file called error-logging.json and updates this file after each error. Types of Errors that may be logged:" belong to the bullet point "Crawler Script:"? If so, can you indent the relevant part?

Overall, the structure of the readme should aid the reader in understanding the content. At the moment, the structure is partially a bit confusing. So, if you can take a pass and fix everything that you see (and also remove outdated info, update new info, etc.).

**Comment by franciscawijaya** [comment]
Done! I have added the headers, cleaned up the formatting according to what information should be under which so that it would be more coherent.

**Comment by SebastianZimmeck** [comment]
Thanks, @franciscawijaya!

I reformatted it a bit more; mostly to be in line with markdown syntax (e.g., each url is enclosed by <>, indentation, etc.). You can see the changes I made in [this commit](https://github.com/privacy-tech-lab/gpc-web-crawler/commit/5a52d8efb72c81b58ee117d8afbd943c785f94b2).

Can you check that this looks correct to you in terms of the structure and if not, make any modifications necessary.

(I have some content questions but will open a separate issue on that.)

**Comment by franciscawijaya** [comment]
Yup, it looks perfect to me! Seeing the edits helped me familiarized myself with some new github syntax and formatting. Thank you!
